Formula/p/parallel-hashmap.rb

```diff
@@ -1,9 +1,10 @@
 class ParallelHashmap < Formula
   desc "Family of header-only, fast, memory-friendly C++ hashmap and btree containers"
   homepage "https://greg7mdp.github.io/parallel-hashmap/"
-  url "https://github.com/greg7mdp/parallel-hashmap/archive/refs/tags/1.37.tar.gz"
-  sha256 "2ac652be0552fcb53a1163c08c1f28f29f0756594fcc587eebb4d8b363153709"
+  url "https://github.com/greg7mdp/parallel-hashmap/archive/refs/tags/v1.3.12.tar.gz"
+  sha256 "0cc203144321924cfbfcc401f42d8204c0dd24e2760c7a1c091baa16d9777c08"
   license "Apache-2.0"
+  version_scheme 1
   head "https://github.com/greg7mdp/parallel-hashmap.git", branch: "master"
 
```

Comment:
```suggestion

  # Upstream switched from a version format like 1.37 to semantic versions like
  # 1.3.8. We're working around this by checking the "latest" release on GitHub
  # until there are newer versions higher than 1.37 (e.g. 1.38.0, 2.0.0).
  livecheck do
    url :stable
    strategy :github_latest
  end

```

It's technically possible to use the `Git` strategy along with a `strategy` block that filters out certain versions but it could eventually become a problem (e.g., if upstream reaches 1.21.0-1.37.0 before 2.0.0). The easy solution in this case is to simply use `GithubLatest` until there's a newer version above the highest older version (e.g., 1.37.1, 1.38.0, 2.0.0, etc.). If/when that happens, we can remove the `livecheck` block and the default `Git` strategy should work as expected again.

Reply:
So I gather that `version_scheme` is something that you don't actively encourage? :)

I'm happy either way.

Reply:
Oh, so this is *in addition to* the `version_scheme` change.

Reply:
`version_scheme` is fine when used for its intended purpose. That is to say, we only use it when upstream has changed their version scheme [in a way that newer versions appear lower than older versions], as in this case. We don't use it to deal with upstream weirdness like yanked versions, etc.

[There's some history behind this: We previously misused `version_scheme` as a way of avoiding the `stable version should not decrease` audit when downgrading a formula but this was never an approved use. For a long time it was the only way to allow those PRs to pass CI but eventually that was explicitly outlawed and I put in some work to create a way for us to skip the `version` audit on CI when needed (controlled by the `CI-version-downgrade` label). Nowadays, `#version_scheme` should only be used for its intended purpose.]

Reply:
> Oh, so this is _in addition to_ the `version_scheme` change.

Yep, they're both necessary. Bumping the `version_scheme` makes brew understand that 1.3.12 is newer than 1.37 (despite the latter being higher than the former) whereas we need the `livecheck` block to make `brew livecheck` return the correct latest version. Without the `livecheck` block, livecheck uses the `Git` strategy by default and that will always return 1.37 as newest until upstream reaches a higher version like 1.37.1, 1.38.0, 2.0.0, etc.

Reply:
Thanks for the detailed explanation @samford!